draft-ietf-suit-information-model.md

```diff
@@ -965,7 +975,7 @@ Implemented by: [Load-time metadata](#manifest-element-load-metadata)
 
 ### REQ.USE.PAYLOAD: Payload in Manifest Superstructure {#req-use-payload}
 
-It MUST be possible to place a payload in the same structure as the manifest. This MAY place the payload in the same packet as the manifest.
+It MUST be possible to place a payload in the same structure as the manifest. This MAY place the payload in the same packet as the manifest. The total size of the signed manifest, including the payload (if any) MUST fit inside device memory.
```

Comment:
It would be useful to state the reason for this requirement, namely that the signature verification has to happen in one go to avoid ToC-ToU attacks.

Reply:
s/device memory/RAM